requirements.txt

```diff
@@ -1,2 +1,2 @@
-django>=1.8.0
+django>=1.11.0
 six
```

Comment:
Do we still need six here?

Reply:
Yes, we do. We're using functions such as `text_type`, `string_types`, and `advance_iterator`.